src/Compilers/Core/Portable/DiagnosticAnalyzer/DefaultAnalyzerAssemblyLoader.cs

```diff
@@ -17,14 +18,18 @@ internal sealed class DefaultAnalyzerAssemblyLoader : AnalyzerAssemblyLoader
     {
 #if NETCOREAPP
 
-        // Called from a netstandard2.0 project, so need to ensure a parameterless constructor is available.
         internal DefaultAnalyzerAssemblyLoader()
```

Comment:
To @RikkiGibson's other question -- do we need this still? Because I don't quite understand the comment here that we had a net2.0 project calling something #ifdef'ed in a NETCOREAPP TFM?

Reply:
There is a need for a parameterless ctor but I think it can be pulled out of the `#if` now and just have a single one.